pyfar/constants/medium_attenuation.py

```diff
@@ -0,0 +1,254 @@
+        +0.1068*np.exp(-3352/T) * (f_rN + (f**2/f_rN))**(-1)))
+
+    alpha = pf.FrequencyData(
+        air_attenuation, frequencies=frequencies)
+
+    m = alpha / (10*np.log10(np.exp(1)))
+
+    # calculate accuracy
+    accuracy = _calculate_accuracy(
+        h, temperature, atmospheric_pressure, frequencies, alpha.freq.shape)
+
+    return alpha, m, accuracy
+
+
+def _calculate_accuracy(
+        concentration_water_vapour, temperature, atmospheric_pressure,
+        frequencies, shape):
+    """Calculate the accuracy of the air attenuation calculation.
+
+    Parameters
+    ----------
+    concentration_water_vapour : float, array_like
+        Molar concentration of water vapor as a percentage.
+    temperature : float, array_like
+        Temperature in degree Celsius.
+    atmospheric_pressure : float, array_like
+        Atmospheric pressure in pascal.
+    frequencies : float, array_like
+        Frequency in Hz.
+    shape : tuple
+        Shape of the output.
+
+    Returns
+    -------
+    accuracy : :py:class:`~pyfar.classes.FrequencyData`
+        accuracy of the results according to the standard:
+
+            ``10``, +/- 10% accuracy
+                - molar concentration of water vapour: 0.05% to 5 %.
+                - air temperature: 253.15 K to 323.15 (-20 °C to +50°C)
+                - atmospheric pressure: less than 200 000 Pa (2 atm)
+                - frequency-to-pressure ratio: 4 x 10-4 Hz/Pa to 10 Hz/Pa.
+
+            ``20``, +/- 20% accuracy
+                - molar concentration of water vapour: 0.005 % to 0.05 %,
+                  and greater than 5%
+                - air temperature: 253.15 K to 323.15 (-20 °C to +50°C)
+                - atmospheric pressure: less than 200 000 Pa (2 atm)
+                - frequency-to-pressure ratio: 4 x 10-4 Hz/Pa to 10 Hz/Pa.
+
+            ``50``, +/- 50% accuracy
+                - molar concentration of water vapour: less than 0.005%
+                - air temperature: greater than 200 K (- 73 °C)
+                - atmospheric pressure: less than 200 000 Pa (2 atm)
+                - frequency-to-pressure ratio: 4 x 10-4 Hz/Pa to 10 Hz/Pa.
+
+            ``-1``, no valid result
+                else.
+    """
+    # broadcast inputs
+    atmospheric_pressure = np.broadcast_to(atmospheric_pressure, shape)
+    h_water_vapor = np.broadcast_to(concentration_water_vapour, shape)
+    frequency_pressure_ratio = frequencies/atmospheric_pressure
+    accuracy = np.zeros(shape) - 1
+
+    # atmospheric pressure < 200 kPa
+    atm_mask = atmospheric_pressure < 200000
+    # frequency-to-pressure ratio: 4 x 10-4 Hz/Pa to 10 Hz/Pa
+    freq2pressure_mask = (4e-4 <= frequency_pressure_ratio) & (
+        frequency_pressure_ratio <= 10)
+    common_mask = atm_mask & freq2pressure_mask
+
+    # molar concentration of water vapour: 0% to 100%
+    vapor_com = (0 <= h_water_vapor) & (h_water_vapor <= 100)
```

Comment:
What do you think of replacing this with an error message, if the value is outside 0-100?

Reply:
This would also allow removing ``& vapor_com`` everywhere below